api/src/main/java/io/strimzi/api/kafka/model/KafkaBridgeSpec.java

```diff
@@ -0,0 +1,195 @@
+import io.fabric8.kubernetes.api.model.ResourceRequirements;
+import io.strimzi.api.kafka.model.template.KafkaBridgeTemplate;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.strimzi.crdgenerator.annotations.Minimum;
+import io.sundr.builder.annotations.Buildable;
+import lombok.EqualsAndHashCode;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = false,
+        builderPackage = "io.fabric8.kubernetes.api.builder"
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({
+        "replicas", "image", "http", "kafka", "consumer",
+        "producer", "resources", "jvmOptions", "logging",
+        "metrics", "livenessProbe", "readinessProbe", "template"})
+@EqualsAndHashCode
+public class KafkaBridgeSpec implements UnknownPropertyPreserving, Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    private int replicas;
+
+    private String image;
+    private KafkaBridgeHttpConfig http;
+    private KafkaBridgeConfigurationSpec kafka;
+    private KafkaBridgeConsumerSpec consumer;
+    private KafkaBridgeProducerSpec producer;
+    private ResourceRequirements resources;
+    private JvmOptions jvmOptions;
+    private Logging logging;
+    private Map<String, Object> metrics;
+    private Probe livenessProbe;
+    private Probe readinessProbe;
+    private Map<String, Object> additionalProperties = new HashMap<>(0);
+    private KafkaBridgeTemplate template;
+
+    @Description("The number of pods in the `Deployment`.")
+    @Minimum(0)
+    @JsonProperty(required = true)
+    public int getReplicas() {
+        return replicas;
+    }
+
+    public void setReplicas(int replicas) {
+        this.replicas = replicas;
+    }
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    @Description("**Currently not supported** The Prometheus JMX Exporter configuration. " +
+            "See {JMXExporter} for details of the structure of this configuration.")
+    public Map<String, Object> getMetrics() {
+        return metrics;
+    }
+
+    public void setMetrics(Map<String, Object> metrics) {
+        this.metrics = metrics;
+    }
+
+    @Description("**Currently not supported** Logging configuration for Mirror Maker.")
+    @JsonInclude(value = JsonInclude.Include.NON_NULL)
+    public Logging getLogging() {
+        return logging;
+    }
+
+    public void setLogging(Logging logging) {
+        this.logging = logging;
+    }
+
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+    @Description("**Currently not supported** JVM Options for pods")
+    public JvmOptions getJvmOptions() {
+        return jvmOptions;
+    }
+
+    public void setJvmOptions(JvmOptions jvmOptions) {
+        this.jvmOptions = jvmOptions;
+    }
+
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    @Description("**Currently not supported** Resource constraints (limits and requests).")
```

Comment:
Why is this not supported? You should make it supported since it should be very easy.